CLAUDE.md

```
@@ -0,0 +1,227 @@
+# Auth
+npm run auth         # Regenerate Better-Auth tables
+
+# Formatting
+npm run fmt          # Auto-format with Prettier
+```
+
+## Architecture
+
+### Monorepo Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)